src/hashtable.c

```diff
@@ -1934,6 +1935,11 @@ size_t hashtableScanDefrag(hashtable *ht, size_t cursor, hashtableScanFunction f
 
 /* --- Iterator --- */
 
+#define HASHTABLE_ITER_UNINITIALIZED -1
+#define HASHTABLE_ITER_PRIMARY_TABLE 0
+#define HASHTABLE_ITER_REHASH_TABLE 1
```

Comment:
These defines are unnecessary and create an inconsistency with the rest of the code.  If we want to create theses (which I don't recommend) the entire file should be refactored to use them consistently.